MAINTAINERS

```diff
@@ -0,0 +1,140 @@
+
+1. A GitHub account may be given in square brackets after an e-mail address.
+
+2. Patches should generally be submitted as GitHub pull requests (more details
+in documentation/github.md). Therefore, please do NOT send patches to the
+e-mail addresses below. Maintainers normally are subscribed to GitHub
+notifications, so they should see your pull requests. If you need to 'ping'
+some maintainer, just mention his/her GitHub account in a comment in the pull
+request.
+
+2. Bug reports and questions should be posted to the GitHub project as well
+(use the "Issues" tab).
+
+[1] https://www.kernel.org/doc/linux/MAINTAINERS
+[2] https://github.com/OP-TEE/optee_os
+
+----------
+
+ARM Foundation FVP
+M:	Linaro <op-tee@linaro.org>
+S:	Maintained
+F:	core/arch/arm/plat-vexpress/
+
+ARM Juno
+M:	Linaro <op-tee@linaro.org>
+S:	Maintained
+F:	core/arch/arm/plat-vexpress/
+
+Atmel ATSAMA5D2-XULT
+M:	Akshay Bhat <akshay.bhat@timesys.com> [@nodeax]
+S:	Maintained
+F:	core/arch/arm/plat-sam/
+
+HiSilicon D02
+M:	Linaro <op-tee@linaro.org>
+S:	Maintained
+S:	core/arch/arm/plat-d02/
+
+HiSilicon HiKey (Kirin 620), HiKey960 (Kirin 960)
+M:	Linaro <op-tee@linaro.org>
+S:	Maintained
+F:	core/arch/arm/plat-hikey/
+
+HiSilicon Poplar (Hi3798C V200)
+M:	Linaro <op-tee@linaro.org>
+S:	Maintained
+F:	core/arch/arm/plat-poplar/
+
+Marvell Armada 70x0, Armada 80x0, Armada 3700
+M:	Kevin Peng <kevinp@marvell.com> [@kevin-peng]
+S:	Maintained
+F:	core/arch/arm/plat-marvell/
+
+MediaTek MT8173 EVB
+M:	Linaro <op-tee@linaro.org>
+S:	Maintained
+F:	core/arch/arm/plat-mediatek/
+
+NXP LS1021A, LS1043A-RDB, LS1046A-RDB
+M:	Sumit Garg <sumit.garg@nxp.com> [@b49020]
+S:	Maintained
+F:	core/arch/arm/plat-ls/
+
+NXP LS1012A-RDB
+M:	Pankaj Gupta <pankaj.gupta@nxp.com> [@pangupta]
+S:	Maintained
+F:	core/arch/arm/plat-ls/
+
+NXP (Freescale) i.MX6 Quad SABRE Lite, i.MX6 Quad SABRE SD
+M:	Yan Yan <yan.yan@windriver.com> [@yanyan-wrs]
+M:	Feng Yu <Yu.Feng@windriver.com>
+S:	Maintained
+F:	core/arch/arm/plat-imx/
+
+NXP (Freescale) i.MX6 UltraLite EVK, i.MX6 ULL EVK, i.MX6 SX Sabreauto, i.MX7 Dual SabreSD
+M:	Peng Fan <peng.fan@nxp.com> [@MrVan]
+S:	Maintained
+F:	core/arch/arm/plat-imx/
+
+QEMU (32 and 64 bits)
+M:	Linaro <op-tee@linaro.org>
+S:	Maintained
+F:	core/arch/arm/plat-vexpress/
+
+Raspberry Pi3
+M:	Linaro <op-tee@linaro.org>
+S:	Maintained
+F:	core/arch/arm/plat-rpi3/
+
+Renesas RCAR
+M:	Volodymyr Babchuk <vlad.babchuk@gmail.com> [@lorc]
+S:	Maintained
+F:	core/arch/arm/plat-rcar/
+
+Rockchip RK322X
+M:	Rockchip <op-tee@rock-chips.com>
+S:	Maintained
+F:	core/arch/arm/plat-rockchip/
+
+Spreadtrum SC9860
+M:	Aijun Sun <aijun.sun@spreadtrum.com>
+S:	Maintained
+F:	core/arch/arm/plat-sprd/
+
+STMicroelectronics b2260-h410, b2120-h310/h410
+M:	Linaro <op-tee@linaro.org>
+S:	Maintained
+F:	core/arch/arm/plat-stm/
+
+Texas Instruments DRA7xx
+M:	Harinarayan Bhatta <harinarayan@ti.com> [@HarinarayanaBhatta]
+S:	Maintained
+F:	core/arch/arm/plat-ti/
+
+Texas Instruments AM43xx
```

Comment:
Harinarayan is no longer a maintainer for these platforms, you can assign all plat-ti to me here.

Texas Instruments AM43xx, AM57xx, DRA7xx

Other than that,

Acked-by: Andrew F. Davis <afd@ti.com>